### PR TITLE
Add docs to Element.performRebuild

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@HansMuller

File: packages/flutter/lib/src/widgets/framework.dart [hunk 1]
```diff
@@ -1411,7 +1411,12 @@ abstract class BuildableElement extends Element {
     assert(!_dirty);
   }
 
-  /// Called by rebuild() after the appropriate checks have been made.
+  /// Called by [rebuild] after [rebuild] has checked that this element indeed
+  /// needs to build and is still active.
+  ///
+  /// This function is called if this element is marked as needing to be built.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/framework.dart]: `(see [markNeedsBuild])`

File: packages/flutter/lib/src/widgets/framework.dart [hunk 2]
```diff
@@ -1411,7 +1411,12 @@ abstract class BuildableElement extends Element {
...
+  /// For example, if setState() is called on its associated [State] object or
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/framework.dart]: `[State.setState]`

**Comment by Hixie** [comment]
LGTM

**Pull request merged**